README.md

```diff
@@ -99,13 +101,13 @@ optional arguments:
                         Default - my-languages.so
 ```
 
-### Building the .so files
+### Building the files
 
 _Please note that building the libraries has been tested under a macOS Mojave with Apple LLVM version 10.0.1 (clang-1001.0.46.4)_
 
 _Please check out our Linux specific instructions [here](https://github.com/autosoft-dev/tree-sitter-docker)_
 
-Once this library is installed it gives you a command line utility to download and compile tree-sitter .so files with ease. As an example - 
+Once this library is installed it gives you a command line utility to download and compile tree-sitter .so or .dll files (depending upon the OS) with ease. As an example -
 
```

Comment:
Create libs will try to git clone first in `/tmp` directory. But that won't be there in Windows. I believe it will need to be fixed also. No?

Reply:
> Create libs will try to git clone first in /tmp directory. But that won't be there in Windows. I believe it will need to be fixed also. No?

```python
TEMP_DOWNLOAD_PATH = "/tmp/tree-sitter-repos" if _which_os() in ["linux", "darwin"] else "vendors"
```
I think it creates a folder named `vendors` and creates library in that. The above line is from `create_libs.py` file.

Reply:
Right, it does. I forgot that I had put it there :) 

However, I see a problem. In case of Linux / Mac it will clone those repos inside /tmp and thus no one will care, but in case of windows it will create a directory called `vendors` inside the present workspace and will clone them there and that people will notice. 

I guess we need to add clean up code if the `.dll` generation is a success. I hope `shtuils.rmtree` works for windows as well to delete an entire directory with all its contents. 

What do you think?

Reply:
> What do you think?
So you mean that we should delete the `vendors` directory?

Reply:
> So you mean that we should delete the vendors directory?

Yes, once we are done, despite success or failure, we should delete it. So that we leave the workspace clean